openmicroscopy/openmicroscopy
# wsgi-args in config

This PR allows pre-setting wsgi-args for gunicorn via bin/omero config as requested in https://github.com/openmicroscopy/management_tools/pull/77#issuecomment-158105212

to test:
 - ``bin/omero config set omero.web.wsgi_args -- '--forwarded-allow-ips=127.0.0.1'``
 - ``bin/omero web start --wsgi-args="--foo"``
 - ``--forwarded-allow-ips`` should overwrite ``--wsgi-args``, check processes for additional args ``ps aux | grep gunicorn``
```
ola             50393   0.0  0.2  2515720  15968   ??  S     4:26pm   0:00.07 /usr/local/Cellar/python/2.7.10_2/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python /usr/local/bin/gunicorn -D -p /Users/ola/OMERO/ome.git/dist/var/django.pid --bind 127.0.0.1:4080 --workers 5 --worker-connections 1000 --max-requests 0 --forwarded-allow-ips=127.0.0.1 omeroweb.wsgi:application
```

cc: @manics 


**Note: change of property name from "omero.web.gunicorn.wsgi_args" to "omero.web.wsgi_args"**
